server/common/models/booking.py

```diff
@@ -0,0 +1,28 @@
+from typing import TYPE_CHECKING
+
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from server.database.config import Base
+from datetime import datetime
```

Comment:
Não vou solicitar uma review pra isso, mas é importante saber. Existem três tipos de import em python: import de bibliotecas built-in do python, import de bibliotecas de terceriros e import de módulos locais do projeto. Os imports têm que ser feitos nessa ordem, e datetime é uma biblioteca padrão do Python. Portanto, a ordem correta dos imports seria

```py
from datetime import datetime
from typing import TYPE_CHECKING

from sqlalchemy.orm import Mapped, mapped_column, relationship

from server.database.config import Base
```

Isso também está especificado na [PEP 8](https://peps.python.org/pep-0008/). O bizu no Pycharm é escrever o nome sem importar e depois fazer ALT + ENTER, que o Pycharm faz o import e coloca ele automaticamente no lugar certo.